### PR TITLE
Feat/showmore

### PR DESCRIPTION
**Pull request by victoriatomzik**

### CODE REVIEW

File: src/Organisms/ShowMore/ShowMore.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  const onShowMoreClick = useCallback(
+    (e) => {
+      onShowMore(e);
+      setShowMoreClicked(true);
+    },
+    [onShowMore],
+  );
+
+  // @ts-ignore
```
Comment (kmelkon) [inline src/Organisms/ShowMore/ShowMore.tsx:43]: this is not a good sign usually :D what's it complaining about?

File: src/Organisms/ShowMore/useOverflow.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { useState, useLayoutEffect } from 'react';
+import { throttle } from 'lodash';
```
Comment (kmelkon) [inline src/Organisms/ShowMore/useOverflow.ts:2]: I'm pretty sure there's a way to throttle without adding a new dependancy.

**Review by kmelkon — CHANGES_REQUESTED** [review]

**Comment by kmelkon** [comment]
Looks good! just left a couple of comments

File: src/Organisms/ShowMore/ShowMore.tsx (hunk 1 above)
Comment (victoriatomzik) [reply src/Organisms/ShowMore/ShowMore.tsx:43]: dude... idek
TS2322: Type 'ReactNode' is not assignable to type 'never'. Type 'undefined' is not assignable to type 'never'.

File: src/Organisms/ShowMore/useOverflow.ts (hunk 2 above)
Comment (victoriatomzik) [reply src/Organisms/ShowMore/useOverflow.ts:2]: ;_;

File: src/Organisms/ShowMore/ShowMore.tsx (hunk 1 above)
Comment (kmelkon) [reply src/Organisms/ShowMore/ShowMore.tsx:43]: hmm we can look into it later but usually TS throws an error cause there's an underlying issue

**Review by alvelid — APPROVED** [review]

**Review by jesperwiner — APPROVED** [review]

**Pull request merged**